File: docs/SDK Documentation/getting-started/create-new-context-request/dotnet/prefetchedData.cs
    var config = new ContextConfig()
        .SetUnit("session_id", "5ebf06d8cb5d8137290c4abb64155584fbdb64d8");
    var context = await _abSdk.CreateContextAsync(config);

    var anotherContextConfig = new ContextConfig()
        .SetUnit("session_id", "5ebf06d8cb5d8137290c4abb64155584fbdb64d9"); // a unique id identifying the other user

    var anotherContext = _abSdk.CreateContextWith(anotherContextConfig, context.GetContextData());

File: docs/SDK Documentation/getting-started/create-new-context-request/php/prefetchedData.php
$contextConfig = new ContextConfig();
$contextConfig->setUnit('session_id', 'session_id5ebf06d8cb5d8137290c4abb64155584fbdb64d8'); // a unique id identifying the user

$context = $sdk->createContext($contextConfig);

$anotherContextConfig = new ContextConfig();
$anotherContextConfig->setUnit('session_id', 'session_id5ebf06d8cb5d8137290c4abb64155584fbdb64d8'); // a unique id identifying the user

$anotherContext = $sdk->createContextWithData($anotherContextConfig, $context->getContextData());
